workflows/files/ZWEWRF02.xml

```diff
@@ -79,6 +79,36 @@ MOUNT FILESYSTEM('${zfsdsn}') +
 TYPE(ZFS) MODE(RDWR) +
 PARM('AGGRGROW') +
 MOUNTPOINT('${zfsmount}')
+/*</inlineTemplate>
+            <submitAs maxRc="8">JCL</submitAs>
+            <maxLrecl>80</maxLrecl>
+        </template>
+    </step>
+  <step name="chown" optional="false">
+        <title>Change ownership</title>
+        <description>This step changes the ownership and mode of ZOWE USS directory so the zwe command can be called.</description>
```

Comment:
This step changes the ownership and mode of the ZOWE USS directory so the zwe command can be called.